script.js

```diff
@@ -1,11 +1,23 @@
 // Create an array called `colors` that contains three colors.
+// var colors = ["red", "yellow", "green"]
 
 // Log the last item in the array to the console.
+// console.log(colors.slice(2))
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+// for (i in colors){
+// 	console.log("My favoritve color is " + colors[i])
+// }
 // Create an `instructor` object that contains three key-value pairs.
+// var instructor = {
+// 	name:"Adrian",
+// 	age: "30",
+// 	hairColor: "brown"
+// }
 
 // Add a `facial-hair` property to `instructor`.
+// instructor.facialhair = "big bushy beard"
```

Comment:
The question was asking how to add a property specifically spelled as `facial-hair` to the object. So really it's asking "How do I add a property that contains a hyphen to an object?".